en/di-extensions.texy

```diff
@@ -67,6 +67,13 @@ $builder->addDefinition($this->prefix('articles'))
 	->addSetup('setLogger', ['@logger']);
 \--
 
+If we need to rename a service, we can create an alias with its original name to maintain backward compatibility. Similarly
+Nette does it for e.q. `routing.router`, which is also available under the earlier name` router`.
```

Comment:
Cítím v tom Google Translate :-)  Což je super!

Akorát typo, uvozovka před router.

Reply:
I feel the power of GT :smirk: